RickMorty/app/src/main/java/com/tolganacar/rickmorty/viewmodel/FeedViewModel.kt

```diff
@@ -0,0 +1,48 @@
+package com.tolganacar.rickmorty.viewmodel
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.tolganacar.rickmorty.model.Character
+import com.tolganacar.rickmorty.model.RMCharacterResponseModel
+import com.tolganacar.rickmorty.service.RickMortyAPIService
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.observers.DisposableSingleObserver
+import io.reactivex.schedulers.Schedulers
+
+class FeedViewModel: ViewModel() {
+    val characterList = MutableLiveData<List<Character>>()
+    val errorMessage = MutableLiveData<Boolean>()
```

Comment:
`errorMessage` yerine `shouldShowErrorMessage` diyebilirsin. Boolean değişkenlerde genellikle isEmpty, shouldShow gibi soru belirten değişken isimleri kullanılıyor. 

is-XXX / should-XXX veya aklına gelen diğer soru kalıpları.